onboarding/src/Components/EmployeeProfile/FormsSentTableRow.js

```diff
@@ -42,34 +43,44 @@ function FormsSentTableRow(props) {
         pages = props.row.pages.map((page, i) => {
             return (
                 <tr key={ i } style={ toggleObj.style }>
-                    <td colSpan="5">
+                    <td className="table__data" colSpan="1">
```

Comment:
`colSpan = 1` is common and default value in general -
- why is it necessary here?